include/libpmemobj++/experimental/radix_tree.hpp

```diff
@@ -616,6 +616,41 @@ radix_tree<Key, Value, BytesView>::radix_tree(InputIt first, InputIt last)
 		emplace(*it);
 }
 
+/* Copy constructor, description todo */
+template <typename Key, typename Value, typename BytesView>
+radix_tree<Key, Value, BytesView>::radix_tree(const radix_tree &m)
+{
+	check_pmem();
+	check_tx_stage_work();
```

Comment:
It would be good to also test this. I would suggest making this file https://github.com/pmem/libpmemobj-cpp/blob/master/tests/vector_ctor_exceptions_notx/vector_ctor_exceptions_notx.cpp generic (and move to tests/container_generic) and then using it to test both concurrent_map and radix tree.